src/Glpi/Form/QuestionType/QuestionTypeDateTime.php

```diff
@@ -310,7 +315,7 @@ public function renderAdministrationOptionsTemplate(?Question $question): string
                 {# Both date and time can be checked at the same time, but one of them must be checked #}
                 function handleDateAndTimeCheckbox_{{ rand }}(input) {
                     const isChecked = $(input).is(':checked');
-                    const otherInput = $('input[onchange^="handleDateAndTimeCheckbox_{{ rand }}"]:not([name="' + input.name + '"])');
+                    const otherInput = $(input).parent().parent().find('input[onchange^="handleDateAndTimeCheckbox_{{ rand }}"]:not([name="' + input.name + '"])');
```

Comment:
Fix an unrelated bug found accidentally thanks to the newly added E2E tests.
This function was not targeting the right container and would impact checkboxes from others questions instead of only targeting its parent question.